src/Components/Web/src/Forms/InputExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    static class InputExtensions
```

Comment:
```suggestion
    internal static class InputExtensions
```

Reply:
Mind if I ask the reason for this suggestion? I didn't use an access modifier because I couldn't see a use for the extension method outside of the `Components.Forms` namespace.

Reply:
We specify the visibility since it's the least ambiguous See https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md#c-coding-style